public/claimlist.php

```diff
@@ -192,7 +192,11 @@
             echo gameAvatar($claim);
             echo "</td>";
             echo "<td>" . ($claim['ClaimType'] == ClaimType::Primary ? ClaimType::toString(ClaimType::Primary) : ClaimType::toString(ClaimType::Collaboration)) . "</td>";
-            echo "<td>" . ($claim['SetType'] == ClaimSetType::NewSet ? ClaimSetType::toString(ClaimSetType::NewSet) : ClaimSetType::toString(ClaimSetType::Revision)) . "</td>";
+            echo "<td class='text-2xs whitespace-nowrap [word-spacing:.2em]'>"
+                . ($claim['SetType'] == ClaimSetType::NewSet ?
+                    ('💡 <b>' . ClaimSetType::toString(ClaimSetType::NewSet) . '</b>') :
```

Comment:
Similar to my other comment, if we are indeed going to make these bold we should use `<span>` tags for better flexibility and accessibility, however I'm not sure this is the message we want to send from a UX POV.